### PR TITLE
Update for patch release

### PR DESCRIPTION
**Pull request by nturinski**
So I didn't do a release branch for this one because the only other change besides the MongoDB driver updates were for the timestamps in the output channel.  I figure when we do testing, we'll just keep an eye out for funky timestamping.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.11.1 - 2019-10-16
+### Added
+- Update mongodb package to 3.3.2
```
Comment (EricJizbaMSFT) [inline CHANGELOG.md]: As discussed offline - would like to know what functional changes this has for the user. Right now it just sounds like engineering work, which we don't typically put in the changelog

Comment (nturinski) [reply CHANGELOG.md]: @fiveisprime Could we discuss what exactly updating the mongodb package fixed for the customer either offline or in a comment in this thread?

**Comment by nturinski** [comment]
- Retryable Writes
- DNS Seedlist Support
- Change Streams
- Sessions

According to https://github.com/mongodb/node-mongodb-native/blob/master/CHANGES_3.0.0.md#sessions

**Comment by nturinski** [comment]
Closing in favor of 0.12.0 release

**Pull request closed without merging**